config.json

```diff
@@ -1135,6 +1135,24 @@
           "strings"
         ]
       },
+      {
+        "slug": "killer-sudoku-helper",
+        "name": "Killer Sudoku Helper",
+        "uuid": "8c45a47d-b3e3-484c-a124-90136ec838fd",
+        "practices": [
+          "constructors",
```

Comment:
> One change I have made is to use method overloading for the implementations as `excludes` is optional. I have tagged the exercise `constructors` as that's the closest to method overloading I could find. Please let me know if this is not the focus of the exercise.

I think this isn't the way to go, instead I think the track could benefit from a concept for method overloading. Tagging it with `constructors` is a bit misleading and will cause confusion as the `KillerSudokuHelper` class doesn't even have any constructors.

Reply:
Should I add the concept pages as part of this PR?